docs/wfm-stitching.ipynb

```diff
@@ -0,0 +1,619 @@
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[91.03, 156.4, 217.18, 269.97, 322.74, 375.0],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=76.76 * deg,\n",
+    "        distance=7.1 * meter,\n",
+    "        name=\"WFM2\",\n",
+    "    ),\n",
+    "    tof.Chopper(\n",
+    "        frequency=56 * Hz,\n",
+    "        open=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[74.6, 139.6, 194.3, 245.3, 294.8, 347.2],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[95.2, 162.8, 216.1, 263.1, 310.5, 371.6],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=62.40 * deg,\n",
+    "        distance=8.8 * meter,\n",
+    "        name=\"Frame-overlap 1\",\n",
+    "    ),\n",
+    "    tof.Chopper(\n",
+    "        frequency=28 * Hz,\n",
+    "        open=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[98.0, 154.0, 206.8, 254.0, 299.0, 344.65],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[134.6, 190.06, 237.01, 280.88, 323.56, 373.76],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=12.27 * deg,\n",
+    "        distance=15.9 * meter,\n",
+    "        name=\"Frame-overlap 2\",\n",
+    "    ),\n",
+    "    tof.Chopper(\n",
+    "        frequency=7 * Hz,\n",
+    "        open=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[30.0],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        close=sc.array(\n",
+    "            dims=[\"cutout\"],\n",
+    "            values=[140.0],\n",
+    "            unit=\"deg\",\n",
+    "        ),\n",
+    "        phase=0 * deg,\n",
+    "        distance=22 * meter,\n",
+    "        name=\"Pulse-overlap\",\n",
+    "    ),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd6e1399-7f05-4d5c-83e3-7d249d9e0a61",
+   "metadata": {},
+   "source": [
+    "## Detector set-up\n",
+    "\n",
+    "We add a detector 32 meters from the source."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a377e75c-51a5-4994-af9c-92139d27bee3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detectors = [\n",
+    "    tof.Detector(distance=32.0 * meter, name=\"detector\"),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fcb52d39-56d6-4cfb-8bcd-be9f45f4cc70",
+   "metadata": {},
+   "source": [
+    "## Find WFM frame edges\n",
+    "\n",
+    "To compute the frame edges, we use one of the openings in the WFM choppers are a time,\n",
```

Comment:
I didn't get this sentence. Especially `are a time` part.

Reply:
Because it should be "we use one of the openings in the WFM choppers at a time" ;-)